README.md

```diff
@@ -270,6 +276,31 @@ set by the [Proselint tool](http://proselint.com/). Since GatorGrader can run an
 arbitrary command and check its error code, it is also possible to integrate it
 with a wide variety of other linters, code formatters, and testing tools.
 
+Instructors may at times need to see a full list of checks to have a better understanding
+and therefore, we feel that it is important to know that there is an easy way to do this.
+This will be a command line and therefore, you can type
```

Comment:
Could we rephrase this sentence to make it more concise and avoid starting two sentence with "this" back to back?